packages/cli-kit/src/private/node/session/exchange.ts

```diff
@@ -84,8 +84,12 @@ export async function exchangeCustomPartnerToken(token: string): Promise<{access
     setLastSeenUserIdAfterAuth(userId)
     setLastSeenAuthMethod('partners_token')
     return {accessToken, userId}
-  } catch (error) {
-    throw new AbortError('The custom token provided is invalid.', 'Ensure the token is correct and not expired.')
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
+    throw new AbortError(
+      'The custom token provided is invalid.',
+      `Ensure the token is correct and not expired.${error.message}`,
```

Comment:
Mistake?

Reply:
yep, i was testing something else, i'll remove this